modules/ducktests/tests/setup.py

```diff
@@ -14,46 +14,22 @@
 # limitations under the License.
 
 import re
-import sys
-
 from setuptools import find_packages, setup
-from setuptools.command.test import test as TestCommand
 
-version = ''
+
 with open('ignitetest/__init__.py', 'r') as fd:
     version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', fd.read(), re.MULTILINE).group(1)
 
 
-class PyTest(TestCommand):
-    user_options = [('pytest-args=', 'a', "Arguments to pass to py.test")]
-
-    def initialize_options(self):
-        TestCommand.initialize_options(self)
-        self.pytest_args = []
-
-    def finalize_options(self):
-        TestCommand.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
-
-    def run_tests(self):
-        # import here, cause outside the eggs aren't loaded
-        import pytest
-        print(self.pytest_args)
-        errno = pytest.main(self.pytest_args)
-        sys.exit(errno)
-
-
 # Note: when changing the version of ducktape, also revise tests/docker/Dockerfile
 setup(name="ignitetest",
       version=version,
       description="Apache Ignite System Tests",
       author="Apache Ignite",
       platforms=["any"],
       license="apache2.0",
-      packages=find_packages(),
+      packages=find_packages(exclude=["ignitetest.tests", "ignitetest.tests.*"]),
       include_package_data=True,
-      install_requires=["ducktape==0.7.8", "requests==2.20.0"],
-      tests_require=["pytest", "mock", "monotonic"],
-      cmdclass={'test': PyTest}
+      install_requires=["ducktape==0.7.8", "requests==2.22.0"],
+      tests_require=["pytest", "mock", "monotonic"]
 )
```

Comment:
As for me, monotonic is install_required, and pytest and mock is not needed at all

Reply:
fixed